.rubocop.yml

```diff
@@ -22,3 +22,6 @@ Style/StringLiterals:
 Style/HashSyntax:
   EnforcedStyle: ruby19
   EnforcedShorthandSyntax: either
+
+Rails/CreateTableWithTimestamps:
+  Enabled: false
```

Comment:
Hmmm. I'm surprised this rubocop rule exists. So could understand an argument that it be disabled. But if this is being done to handle the rubocop failure within the single migration in this PR, then I think it should be disabled in that migration directly rather than globally here. That is add `# rubocop:disable Rails/CreateTableWithTimestamps` to the beginning of the migration and `# rubocop:disable Rails/CreateTableWithTimestamps` to the end.

Reply:
I could go either way - I was surprised it was required and felt disabling was a fine way to go.